source/features/rgh-improve-new-issue-form.tsx

```diff
@@ -18,6 +20,14 @@ function init(): void {
 			Clear cache
 		</button>,
 	);
+	void expectTokenScope('repo').catch(() => {
+		select('#issue_body_template_name')!.before(
+			<div className="flash flash-warn m-2">
+				Your Personal Access Token is either missing, incorrect or expired. Some Refined GitHub features will not work without it.<br/>
```

Comment:
We can be more specific here, maybe with a custom API call to check, in order:

- existence (no call at all)
- validity
- expiration
- lack of expected scope